rocAL/rocAL_pybind/example/new_api/rocAL_api_test.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import random
+from amd.rocal.plugin.pytorch import RALI_iterator
+
+from amd.rocal.pipeline import Pipeline
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import sys
+import cv2
+import os
+
+def main():
+    if  len(sys.argv) < 3:
+        print ('Please pass image_folder cpu/gpu batch_size')
+        exit(0)
+    data_path = sys.argv[1]
+    if(sys.argv[2] == "cpu"):
+        _rali_cpu = True
+    else:
+        _rali_cpu = False
+    batch_size = int(sys.argv[3])
+    num_threads = 1
+    device_id = 0
+    random_seed = random.SystemRandom().randint(0, 2**32 - 1)
+    crop=300
+
+    pipe = Pipeline(batch_size=batch_size, num_threads=num_threads,device_id=device_id, seed=random_seed, rali_cpu=_rali_cpu)
+    local_rank = 0
+    world_size = 1
+    rali_cpu= True
+    rali_device = 'cpu' if rali_cpu else 'gpu'
+    decoder_device = 'cpu' if rali_cpu else 'mixed'
```

Comment:
we don't support mixed devices in rocal and so as padding. Please remove those

Reply:
Removed in the latest commit